Makefile

```diff
@@ -184,8 +184,8 @@ local-setup-cluster: $(KIND) ## Setup local development kind cluster, dependenci
 
 	@echo "local-setup: Check dns operator deployments"
 	$(KUBECTL) get deployments -l app.kubernetes.io/part-of=dns-operator -A
-	@echo "local-setup: Check managedzones"
-	$(KUBECTL) get managedzones -A
+	@echo "local-setup: Check dns providers"
+	$(KUBECTL) get secrets -A
```

Comment:
won't this list unrelated secrets, i.e. all the secrets in the cluster? Could we add a label selector or something, or am I mistaken?

Reply:
:+1: There are labels being added to the secrets so we can use a the same on as we are using for deployments `-l app.kubernetes.io/part-of=dns-operator`

output now:

```
local-setup: Check dns providers                           
NAMESPACE   NAME                                TYPE                   DATA   AGE
dnstest     dns-provider-credentials-aws        kuadrant.io/aws        5      1s
dnstest     dns-provider-credentials-azure      kuadrant.io/azure      3      1s                                     
dnstest     dns-provider-credentials-gcp        kuadrant.io/gcp        4      1s
dnstest     dns-provider-credentials-inmemory   kuadrant.io/inmemory   1      0s
local-setup: Complete!!
```